draft-ietf-httpbis-http2bis.xml

```diff
@@ -1415,6 +1415,16 @@
         <t>
           A list of error codes is included in <xref target="ErrorCodes"/>.
         </t>
+        <t>
+          It is possible that an endpoint will encounter frames that would cause multiple errors. Implementations MAY process
+          multiple errors triggered by a single frame in any order, within the limitations set out in this section. Once an
+          implementation has handled a connection error, no further frame processing will be possible. As a result, connection
+          errors are invariably the last error type handled by an implementation.
```

Comment:
Unfortunately, I think that we can change our minds when we report a connection error.  And we can report stream errors after a connection error.  But we should advise against it.

> The first stream error reported for a given stream prevents any other errors on that stream from being reported.  In comparison, the protocol permits multiple GOAWAY frames, though an endpoint SHOULD report just one type of connection errors unless a graceful shutdown (see $$SECTION) is cut short as a result of a connection error is encountered.